### DISCUSSION TITLE
Tablet behavior messed up when on osu lazer is active window

### DISCUSSION THREAD

**Discussion by giyugg** [ppy/osu#30419] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Sorry for the bad English.

Somehow, when lazer is an active window on windows 11, it messes up my area on my tablet. Idk if it's only me.
In the first part of the video, I demonstrated how my cursor responds when it's not on osu lazer yet (the focus window). After I alt-tab, focusing on lazer, the cursor does not correspond with my area coordinates.

### Screenshots or videos

https://github.com/user-attachments/assets/61d7efd7-1e68-490a-a24f-e9e5814b78a7

### Version

2024.1009.1

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17504805/compressed-logs.zip)

**Comment by smoogipoo** [comment, marked as answer]
Perhaps try enabling/disabling high-precision mouse in the settings.

**Reply by giyugg** [reply]
holy shit you just saved my day! thank you so much
